README.md

```diff
@@ -7,12 +7,12 @@ The only supported Trusted Third Party is currently AWS IAM.
 
```

Comment:
In the README, I would add a bit of motivation for using this tool. I feel like it's not entirely clear.

Reply:
Not a blocker, but more comments in the code would be great.

Reply:
README.md will be improved in another PR